lost/src/test/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImplTest.java

```diff
@@ -203,12 +203,12 @@ public void removeLocationUpdates_shouldUnregisterAllListeners() throws Exceptio
     }
 
     @Test
-    public void setMockMode_shouldUnregisterAllListenersWhenTrue() throws Exception {
+    public void setMockMode_shouldToggleAllListenersWhenTrue() throws Exception {
```

Comment:
The original purpose of this test was to ensure all location listeners were removed from the `LocationManager` so we are not invoking system location services while in mock mode. I think that test is still valid in its original form but maybe we want to update the name to something like `setMockMode_shouldRemoveAllLocationManagerListeners()`.

Perhaps we should add an additional test to verify the correct number of listeners has been set on the `FusedLocationProviderApi` by the client app.